DaemonRenderer/Source/Src/Vulkan/CommandPool.cpp

```diff
@@ -64,46 +59,49 @@ CommandPool::CommandPool(Scheduler& in_scheduler, DAEuint32 const in_queue_famil
 
 VulkanCommandBuffer* CommandPool::RequestCommandBuffer(VkCommandBufferLevel const in_level) noexcept
```

Comment:
This method wouldn't mind some more documentation to explain what is going on.

Reply:
Above the declaration or inside the definition ?

Reply:
Inside the declaration